src/stream/tag.rs

```diff
@@ -226,7 +226,7 @@ impl Encoder {
     ///
     /// Note that the plain tag is written, regardless of the original contents. To safely encode a
     /// tag to an MP3 file, use `Encoder::encode_to_path`.
-    pub fn encode(&self, tag: &Tag, mut writer: impl io::Write) -> crate::Result<()> {
+    pub fn encode(&self, tag: &Tag, mut writer: impl io::Write, padding: Option<usize>) -> crate::Result<()> {
```

Comment:
Encoder has a complementary Builder, could you please add the padding argument to that? That way we can keep the API backwards compatible :)